activemq-broker/src/main/java/org/apache/activemq/broker/BrokerService.java

```diff
@@ -152,10 +152,15 @@
     private boolean useShutdownHook = true;
     private boolean useLoggingForShutdownErrors;
     private boolean shutdownOnMasterFailure;
+    private boolean shutdownOnActiveFailure;
```

Comment:
Its a private field no need to duplicate just rename the original field. Just need to keep method compatibility

Reply:
This comment applies generally. No need for me to repeat on every occurance

Reply:
Hmm good point, my brain didn't catch that as my first pass was assuming spring looked at the actual variable names.

Reply:
This was addressed in latest version of PR